crates/m3/src/gadgets/hash/keccak/mod.rs

```diff
@@ -356,9 +398,9 @@ impl RoundBatch {
 		P: PackedFieldIndexable<Scalar = B128> + PackedExtension<B1> + PackedExtension<B8>,
 		PackedSubfield<P, B8>: PackedTransformationFactory<PackedSubfield<P, B8>>,
 	{
-		for (k, pt) in pts.iter().enumerate() {
+		for (k, trace) in pts.iter().enumerate() {
```

Comment:
Not sure if this is clear, but pt is part of pts. (it's the same as functional people x → xs). So either go the whole way, and rename pts, or revert. I am ok with either, however, for further PRs I would suggest reducing churn (things like renaming, refactoring, or even changing whitespaces) if possible and if not, the next best thing is to move out to another PR. Good news this just got easier with graphite.